uploadservice/src/main/java/net/gotev/uploadservice/NotificationAction.java

```diff
@@ -28,8 +30,12 @@ public String getText() {
         return text;
     }
 
-    public PendingIntent getClickIntent() {
-        return clickIntent;
+    public final PendingIntent getPendingIntent(Context context) {
+        if (clickIntent == null) {
+            return PendingIntent.getBroadcast(context, 0, new Intent(), PendingIntent.FLAG_UPDATE_CURRENT);
```

Comment:
That has to be changed as well to be able to set pending intent instead of intent.